v8env/src/globals.ts

```diff
@@ -111,6 +112,7 @@ window.fly = {
   data: flyData,
   http: flyHttp,
   Image: Image,
+  runtime,
```

Comment:
I think you need to add that to the fly interface nested in the window interface above.